lib/cocoapods/command/repo/add.rb

```diff
@@ -34,9 +34,9 @@ def validate!
           unless @name && @url
             help! 'Adding a repo needs a `NAME` and a `URL`.'
           end
-          if @name == 'master' || @url =~ %r{github.com[:/]+cocoapods/specs}i
+          if @name == 'master'
             raise Informative,
-                  'To setup the master specs repo, please run `pod setup`.'
+                  'Master repo via git has been replaced with a CDN-based solution.'
```

Comment:
can people still fallback to use master specs repo as a normal repo?

Reply:
Yes - just not named `master`. Do you think that's a problem?

Reply:
probably not, could be scoped specifically to the master specs URL.

Reply:
Come to think of it, there's no reason to reserving anything other that `trunk`.

I'll rewrite it.